docs/process/metrics.md

```diff
@@ -0,0 +1,51 @@
+| `coAuthoredCommits` | The number of commits created with one or more co-authors. | To understand usage patterns of commits made in Desktop. |
+| `branchComparisons` | The number of times a branch is compared to an arbitrary branch. | To understand usage patterns around the compare branches feature. |
+| `defaultBranchComparisons` | The number of times a branch is compared to `master`. | To understand usage patterns around the compare branches feature. |
+| `mergesInitiatedFromComparison` | The number of times a merge is initiated in the `compare` sidebar. | To understand usage patterns around the compare branches feature. |
+| `updateFromDefaultBranchMenuCount` | The number of times the `Branch -> Update From Default Branch` menu item is used. | To understand usage patterns around the compare branches feature. |
+| `mergeIntoCurrentBranchMenuCount` | The number of times the `Branch -> Merge Into Current Branch` menu item is used. | To understand usage patterns around the compare branches feature. |
+| `prBranchCheckouts` | The number of times the user checks out a branch using the PR menu. | To understand usage patterns around the PR checkout menu. |
+| `repoWithIndicatorClicked` | The numbers of times a repo with indicators is clicked on repo list view. | To understand usage patterns around the additional information in the repository list feature. |
+| `repoWithoutIndicatorClicked` | The numbers of times a repo without indicators is clicked on repo list view.  | To understand usage patterns around the additional information in the repository list feature. |
+| `divergingBranchBannerDismissal` | The number of times the user dismisses the diverged branch notification. | To understand usage patterns around the notification of diverging from the default branch feature. |
+| `divergingBranchBannerInitatedMerge` | The number of times the user merges from the diverged branch notification merge CTA button. | To understand usage patterns around the notification of diverging from the default branch feature. |
+| `divergingBranchBannerInitiatedCompare` | The number of times the user compares from the diverged branch notification compare CTA button. | To understand usage patterns around the notification of diverging from the default branch feature. |
+| `divergingBranchBannerInfluencedMerge` | The number of times the user merges from the compare view after getting to that state from the diverged branch notification compare CTA button. | To understand usage patterns around the notification of diverging from the default branch feature. |
+| `divergingBranchBannerDisplayed` | The number of times the diverged branch notification is displayed. | To understand usage patterns around the notification of diverging from the default branch feature. |
+| `active*` | Whether or not the app has been interacted with during the current reporting window. | To identify users who are actively using Desktop versus those who have it open but never interact with it. |
```

Comment:
What's the `*` here for?

Reply:
:art:

> Whether or not the app has been interacted with during the current reporting window.

could be

> Flag indicating whether the app has been interacted with during the current reporting window.

Reply:
The `*` was my signifier that I believed a field was in the wrong category. In source, this field is a measure, but it should have been defined as a dimension.